assets/css/vendor/leaflet.css

```diff
@@ -240,55 +298,38 @@
 	color: #bbb;
 	}
 
-.leaflet-touch .leaflet-bar {
-	-webkit-border-radius: 10px;
-	        border-radius: 10px;
-	}
 .leaflet-touch .leaflet-bar a {
 	width: 30px;
 	height: 30px;
-	}
-.leaflet-touch .leaflet-bar a:first-child {
-	-webkit-border-top-left-radius: 7px;
-	        border-top-left-radius: 7px;
-	-webkit-border-top-right-radius: 7px;
-	        border-top-right-radius: 7px;
-	}
-.leaflet-touch .leaflet-bar a:last-child {
-	-webkit-border-bottom-left-radius: 7px;
-	        border-bottom-left-radius: 7px;
-	-webkit-border-bottom-right-radius: 7px;
-	        border-bottom-right-radius: 7px;
-	border-bottom: none;
+	line-height: 30px;
 	}
 
 
 /* zoom control */
 
-.leaflet-control-zoom-in {
+.leaflet-control-zoom-in,
+.leaflet-control-zoom-out {
 	font: bold 18px 'Lucida Console', Monaco, monospace;
+	text-indent: 1px;
 	}
 .leaflet-control-zoom-out {
-	font: bold 22px 'Lucida Console', Monaco, monospace;
+	font-size: 20px;
 	}
 
 .leaflet-touch .leaflet-control-zoom-in {
 	font-size: 22px;
-	line-height: 30px;
 	}
 .leaflet-touch .leaflet-control-zoom-out {
-	font-size: 28px;
-	line-height: 30px;
+	font-size: 24px;
 	}
 
 
 /* layers control */
 
 .leaflet-control-layers {
-	box-shadow: 0 1px 7px rgba(0,0,0,0.4);
-	background: #f8f8f9;
-	-webkit-border-radius: 5px;
-	        border-radius: 5px;
+	box-shadow: 0 1px 5px rgba(0,0,0,0.4);
+	background: #fff;
+	border-radius: 5px;
 	}
 .leaflet-control-layers-toggle {

```

Comment:
I don't see this issue -- try `scripts/manage.sh collectstatic`.

(Note that without branding colors webpack bundles the images directly but with branding colors the dynamically-compiled SASS refers to the static image location.)